mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -66,6 +66,10 @@
     <string name="mozac_feature_addons_settings_on">On</string>
     <!-- Indicates the add-on is disabled. -->
     <string name="mozac_feature_addons_settings_off">Off</string>
+    <!-- Indicates the add-on is allowed in private browsing mode. -->
+    <string name="mozac_feature_addons_settings_allow_in_private_browsing">Allow in private browsing</string>
+    <!-- Indicates the add-on is allowed in private browsing mode. -->
+    <string name="mozac_feature_addons_settings_run_in_private_browsing">Run in private browsing</string>
```

Comment:
@psymoon  
What's the difference between these two, given that they have the same comment?

Reply:
@flodolo "Run in private browsing" is from the new UX mockup, will eventually replace "Allow in private browsing".

Reply:
Thanks. It would be great to not have folks translate the old version, but I guess it can be avoided.